webapi/src/main/scala/org/knora/webapi/responders/v2/ValuesResponderV2.scala

```diff
@@ -853,6 +852,11 @@ class ValuesResponderV2(responderData: ResponderData) extends Responder(responde
                 _ = if (currentValue.valueContent.valueType != submittedExternalValueType.toOntologySchema(InternalSchema)) {
                     throw BadRequestException(s"Value <$valueIri> has type <${currentValue.valueContent.valueType.toOntologySchema(ApiV2Complex)}>, but the submitted type was <$submittedExternalValueType>")
                 }
+
+                // If a custom value creation date was submitted, make sure it's later than the date of the current version.
```

Comment:
this check did not come to my mind back then when I worked on custom creation date. I trust clients too much I guess that I had not thought they might give a date in the past!

Reply:
I thought that if the user did make that mistake, it would totally mess up the version history of the value, because the version history query depends on the timestamps being in the right order.